CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ For information on changes in released versions of Teku, see the [releases page]
 - The `validator_beacon_node_published_attestation_total`, `validator_beacon_node_published_aggregate_total`,
   `validator_beacon_node_send_sync_committee_messages_total`, `validator_beacon_node_send_sync_committee_contributions_total`
   and `validator_beacon_node_published_block_total` metrics have been replaced by the new `validator_beacon_node_requests_total` metric. An update to the [Teku Dashboard](https://grafana.com/grafana/dashboards/13457) that uses the new metric is available.
+- Deprecated v1 of beacon block methods `/eth2/beacon_chain/req/beacon_blocks_by_root` and `/eth2/beacon_chain/req/beacon_blocks_by_range`.
```

Comment:
I think more accurately we've removed references to those functions, not deprecated their use?